release/tar/linux/opensearch-tar-install.sh

```diff
@@ -20,7 +20,6 @@ bash $OPENSEARCH_HOME/plugins/opensearch-security/tools/install_demo_configurati
 
 ##Perf Plugin
 chmod 755 $OPENSEARCH_HOME/plugins/opensearch-performance-analyzer/pa_bin/performance-analyzer-agent
-chmod -R 755 /dev/shm
```

Comment:
Good removal as it is already removed in onetime script.
This line actually never works because if you run in root the whole script does not work but if you dont run in root that chmod never works.

I should replace it last time when I remove it from onetime script. Thanks.